.github/workflows/push-on-develop.yml

```
@@ -95,8 +102,13 @@ jobs:
 
       - name: Create manifest list and push 📦
         run: |
-          docker buildx imagetools create --tag "${{ env.REGISTRY_IMAGE }}:develop" \
-          $(cat /tmp/digests/*-digest.txt | tr '\n' ' ')
+          DIGESTS=$(for f in /tmp/digests/*-digest.txt; do
+            PLATFORM=$(basename "$f" | sed 's/-digest.txt//')
+            DIGEST=$(cat "$f")
+            echo "${{ env.REGISTRY_IMAGE }}:${PLATFORM}@${DIGEST}"
+          done | tr '\n' ' ')
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)